ubo_app/main.py

```diff
@@ -23,6 +23,10 @@ def main() -> None:
     os.environ['KIVY_NO_FILELOG'] = '1'
     os.environ['KIVY_NO_CONSOLELOG'] = '1'
     os.environ['KCFG_KIVY_EXIT_ON_ESCAPE'] = '0'
+    os.environ['KCFG_GRAPHICS_FBO'] = 'force-hardware'
```

Comment:
@sassanh it would be great to have some explanation of what each env variable does somewhere...